src/NServiceBus.Hosting.Windows/StartableAndStoppable/StartableAndStoppableRunner.cs

```diff
@@ -21,9 +22,13 @@ public Task Start(IMessageSession session)
             var startableTasks = new List<Task>();
             foreach (var startable in wantToRunWhenBusStartsAndStops)
             {
-                var task = startable.Start(session).ThrowIfNull();
-
                 var startable1 = startable;
+                var startableName = startable1.GetType().AssemblyQualifiedName;
+
+                var longRunningTokenSource =
+                    LongRunningWarning($"The start method for {startableName} is taking too long to complete. The endpoint will not start until this operation has completed. Until then messages that arrive in the endpoint's queue will not be processed.");
```

Comment:
How about changing it to "is taking a long time to complete" which I think works better for the intention of the warning, without making it feel like we need to tell them how long is too long.

Also, I'm not sure the third sentence is needed, since we've already said the endpoint won't start until the operation has completed.